src/actions/create.js

```diff
@@ -39,16 +41,25 @@ export function create(config, schema, item = null) {
     body = config.body;
   }
 
+  const isEndpointResolved = options[RESOLVED_ENDPOINT];
+  const endpoint = isEndpointResolved
```

Comment:
those shared parts like `buildRSAAConfig` chould be moved into seperate file, for now maybe schemaConfig.js